### PR TITLE
Add accurate PPU FMA to advanced tab

### PR DESCRIPTION
**Pull request by illusion0001**
Followup for https://github.com/RPCS3/rpcs3/pull/7874 which was missing option from gui

### CODE REVIEW

**Comment by elad335** [comment]
I prefer it in config file for now to hide it from stupid users.
And users should almost never turn it off unless they have a really old cpu.

**Comment by elad335** [comment]
on second thought the "Never use this." description should be enough to do the job.

**Comment by elad335** [comment]
Ah, maybe reword it to "Never disable it." to not invoke confusion for users which may disable it because of it.

**Comment by illusion0001** [comment]
@elad335 fixed

**Comment by AniLeo** [comment]
This should actually be in advanced

**Comment by dio-gh** [comment]
No need to hide what it does from people either, "never disable this" is a terrible description. Can you make it so that it only shows up for people with CPUs that don't have in-hw FMA support?

**Comment by illusion0001** [comment]
I can move it to advanced tab. Not sure about the description? Maybe someone can suggest. The check for fma support isn't possible with my very limited coding knowledge.

**Comment by RainbowCookie32** [comment]
Maybe something like "Enables extra accuracy on FMA instructions, which can be needed by some games. It can impact performance negatively on CPUs without FMA acceleration support"

**Comment by dio-gh** [comment]
> The check for fma support isn't possible with my very limited coding knowledge

If you include `Emu/CPU/CPUTranslator.h` you'll be able to check against `m_use_fma`. I'm not sure if it's a good idea to do it like this though.

If you can check for it like this, the description could be `Improves FMA operations' accuracy at the cost of performance. While disabling it on your CPU might give a decent performance boost, it may also introduce subtle bugs that otherwise do not occur.`

**Comment by illusion0001** [comment]
Moved to advanced below debug console mode

**Comment by illusion0001** [comment]
Didn't notice the updates. Thanks for updating

**Comment by elad335** [comment]
Wtf, it was suppised to be debug tab with the description of "Never disable this".
Now I need to fix this mess.

**Comment by AniLeo** [comment]
No, it isn't, see #7925
